lib/readline.js

```diff
@@ -66,7 +67,7 @@ function Interface(input, output, completer, terminal) {
   if (typeof historySize !== 'number' ||
       isNaN(historySize) ||
       historySize < 0) {
-    throw new TypeError('Argument "historySize" must be a positive number');
+    throw new errors.TypeError('ERR_NON_NEGATIVE_NUMBER');
```

Comment:
I think this is actually a `RangeError`? This looks like a `ERR_INVALID_OPT_VALUE` to me (refs: https://github.com/nodejs/node/pull/11300/files) too, but we can also make the message format of `ERR_NON_NEGATIVE_NUMBER ` a function taking description of the variable that's negative, so we can keep the "historySize" part.

On another note, if we want to keep it as a `ERR_NON_NEGATIVE_NUMBER`, that code is missing documentation.